tools/ZydisFuzzShared.c

```diff
@@ -147,6 +147,67 @@ void ZydisValidateEnumRanges(const ZydisDecodedInstruction* insn,
             " = 0x%016" PRIX64 "\n", (ZyanU64)(value), (ZyanU64)(max));                            \
         abort();                                                                                   \
     }
+#   define ZYDIS_CHECK_MAX ZYDIS_CHECK_ENUM
```

Comment:
I'm not a fan of these macros but this was the only solution i could come up with.